internal/buildops/testdata/mtahtml5/mtaRequiredModuleNotDefined.yaml

```diff
@@ -0,0 +1,16 @@
+ID: mtahtml5
+_schema-version: '2.1'
+version: 0.0.1
```

Comment:
I would add a new test with two modules that one defined as expected and the other is like below

Reply:
this case covers correction done in modules dependencies functionality to provide a proper log error message in case of missing required module, not validation scenario.